cogs/memes.py

```diff
@@ -25,6 +25,11 @@ def __init__(self, bot):
         with open('data/memes/lex.png', 'rb') as f:
             await self.bot.send_file(ctx.message.channel, f)
 
+    @commands.command(pass_context=True)
+    async def collin(self, ctx):
+        """F this"""
+        await self.bot.say("https://gfycat.com/AffectionateGenerousBarebirdbat")
+
     @commands.group(name="smith", pass_context=True)
```

Comment:
is `name="smith"` required here? I don't see any of the other meme commands following this format

Reply:
Good catch; old code. I'll fix that.